### ISSUE TITLE
These are all the files needed for OS X on the GA-Z170X-UD3? Does everything work?

### ISSUE THREAD

**Issue by 0xKiwi** [cnrd/GA-Z170X-UD3-OSX#1]
Dude thank you so much for this man, I might get this board and these files all being together is a godsend. Does everything work on OS X with the mobo?

Thanks!

**Comment by cnrd** [comment]
Yeah I have everything except the USB 3.1 ports working.

I haven't really cleaned up the repo, so some of the stuff may not be useful.

**Comment by 0xKiwi** [comment]
Dude that is aawwwweesomme. I just bought this motherboard. All I have left
is my cpu to buy. Your repo will help out a ton. Thank you so much.
Everything is in the guide file right?

**Comment by cnrd** [comment]
Yeah pretty much, it's in BBCODE.

But replace the USB part with the USB-SSDT.zip (Haven't gotten around to update that part).

EDIT: If you want it better formatted you can find the guide here: http://www.tonymacx86.com/threads/guide-el-capitan-on-the-ga-z170x-ud3.180618/

**Comment by 0xKiwi** [comment]
Gotcha. Thank you!!

**Comment by 0xKiwi** [comment]
Is it okay if the installation USB is prepared with Unibeast? Or is the method in the guide required? I prepared a USB before finding this repo and want to make sure if I do or don't need to redo it.

Thanks!

**Comment by cnrd** [comment]
You can use the Unibeast USB to install.

I just like creating my own USB (Better control than using Unibeast).

You will just have to jump to the part of creating the USB where it shows which files to copy where.

I have updated the guide to include all of the new ways I do things (there are quite a lot of changes), and also restructured the git repo a bit.

**Comment by 0xKiwi** [comment]
Excellent! Thank you so much! I'll let you know how it goes!

**Comment by 0xKiwi** [comment]
~~I think I forgot to realize that I'm not using a GPU. I'm using the embedded graphics. I've been going at this for hours but I'm positive that is my issue. What changes do I need to use the integrated Intel HD 530 graphics? I'm new to this so I realize it was dumb for me to do all of this assuming things like the SSDT and config.plist would work on my embedded graphics when this is configured for an Nvidia GPU correct?~~

**Comment by 0xKiwi** [comment]
Never mind ^

After so many hours of trying, I'm pretty sure my USB was at fault. Not sure why. 

I managed to get it working (from the awesome suggestion of my friend) by putting clover and os x install media directly on the SSD I was planning to install OS X on. I installed it, removed the partition I used to install it, and it all worked like a charm!

All the USB ports work, HDMI works fine, the Headphone port works fine (I just forgot to set output to headphones), I have yet to check if Ethernet works. I will report back on that. But overall thank you so much man! You've made this process 10,000x easier!
